webapp/src/ts/services/language.service.ts

```diff
@@ -0,0 +1,93 @@
+import * as moment from 'moment';
+import { Injectable } from '@angular/core';
+import { CookieService } from 'ngx-cookie-service';
+import { TranslateService } from '@ngx-translate/core';
+import { SettingsService } from './settings.service';
+import { UserSettingsService } from './user-settings.service';
+
+const localeCookieKey = 'locale';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SetLanguageCookieService {
+  constructor(
+    private cookieService:CookieService
+  ) {
+  }
+
+  set(value) {
+    this.cookieService.set(localeCookieKey, value, 365, '/');
+    return value;
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SetLanguageService {
+  constructor(
+    private translateService:TranslateService,
+    private setLanguageCookieService:SetLanguageCookieService,
+  ) {
+  }
+
+  private setDatepickerLanguage(language) {
+    // TODO
```

Comment:
Is this coming in the next PR?

Reply:
Not in next PR. This will come when we add Enketo (that's where the datepicker is used).